lib/greenhouse_io/api/client.rb

```diff
@@ -105,6 +105,15 @@ def create_custom_field(custom_field_hash, on_behalf_of = nil)
       )
     end
 
+    def update_custom_field(custom_field_id, custom_field_hash, on_behalf_of = nil)
```

Comment:
These are the major changes - everything in `spec` is fixing broken fixtures from the main branch.